inst/qml/LSBuffonsneedlemanipulation.qml

```diff
@@ -0,0 +1,132 @@
+	//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick			2.8
+import QtQuick.Layouts	1.3
+import JASP.Controls	1.0
+import JASP.Widgets		1.0
+import JASP.Theme		1.0
+
+Form
+{
+	columns: 1
+
+
+	IntegerField 
+	{ 
+	name: "length"; 
+	label: qsTr("Proportion of needle length to interline distance:")
+	defaultValue: 80
+	afterLabel: qsTr("%")
+	min: 1
+	max: 100
+	}
+
+
+
+	IntegerField   
+	{ 
+		name:			"n"
```

Comment:
```suggestion
		name:			"n"
		id: n
```

you need to add the `id` here so that later you can use `n.value` to set the max bound